src/styles/utils/variables.scss

```diff
@@ -0,0 +1,7 @@
+$column-size: 100px;
+$columns-in-row: 7;
+$gap-between-columns: 1px;
+$gaps-in-row: $columns-in-row - 1;
+$calendar-padding: 10px;
+$weekdays:
+mon 0, tue 101px, wed 202px, thu 303px, fri 404px, sat 505px, sun 606px;
```

Comment:
How did you get these numbers? Would be great to calculate them instead of hardcoding